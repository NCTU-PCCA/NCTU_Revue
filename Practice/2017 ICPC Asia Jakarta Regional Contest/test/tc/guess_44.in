603267618438799485 37
><><<<><>><><><><<<<><<><>>>>><<<>><<
