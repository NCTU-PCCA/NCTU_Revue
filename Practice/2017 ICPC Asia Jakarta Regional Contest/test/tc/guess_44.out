1 603267618438799485 2 603267618438799484 603267618438799483 603267618438799482 3 603267618438799481 4 5 603267618438799480 6 603267618438799479 7 603267618438799478 8 603267618438799477 603267618438799476 603267618438799475 603267618438799474 9 603267618438799473 603267618438799472 10 603267618438799471 11 12 13 14 15 603267618438799470 603267618438799469 603267618438799468 16 17 603267618438799467 603267618438799466
